code/modules/projectiles/guncrafting/attachments/barrels.dm

```diff
@@ -0,0 +1,17 @@
+/obj/item/weapon/gun_attachment/barrel/short
+	name = "short barrel"
+	icon_state = "barrel_short"
+	range = 7
```

Comment:
You just made roundstart tasers worthless in disable mode because the main advantage of them in paprika's time was that they had more range. Guess what this is?!